lambdas/bulk-migrate-vcs/src/main/java/uk/gov/di/ipv/core/bulkmigratevcs/BulkMigrateVcsHandler.java

```diff
@@ -228,6 +253,12 @@ private void migrateIdentity(
                     e,
                     batchId,
                     pageSummary);
+            auditService.sendAuditEvent(
+                    AuditEvent.createWithoutDeviceInformation(
+                            AuditEventTypes.IPV_EVCS_MIGRATION_FAILURE,
+                            configService.getParameter(ConfigurationVariable.COMPONENT_ID),
+                            new AuditEventUser(reportItem.getHashUserId(), "", "", ""),
```

Comment:
I am not sure, If we need to send actual userId or hash one

Reply:
Ah, as above, we should be using the userId